Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -1377,23 +1360,13 @@ function Set-HcxScaledCpuAndMemorySetting {
                 }
             }
         }
-
         Write-Host "$Appliances appliances found."
 
-        Write-Output "Create HCX SSH Session:"
-        $SshSession = New-SSHSession -ComputerName $HcxServer -Credential $HcxAdminCredential -Port 22 -Force
-        $Stream = New-SSHShellStream -SSHSession $SshSession -TerminalName 'PS-SSH' -Columns 0 -Rows 0 -Width 0 -Height 1000 -BufferSize 10000
-
-        Write-Host "Retrieving /Common Percentage"
-        $DiskFileSystemReturn = Invoke-SSHStreamShellCommand -ShellStream $Stream -Command 'df -h | grep "/common"'
-        if ($DiskFileSystemReturn.GetType().Name -eq 'Object[]') { $SlashCommonPercentage = $($DiskFileSystemReturn[0] -Split '\s+')[4] }
-        else { $SlashCommonPercentage = '0%' }
-
-        if ($SlashCommonPercentage -gt $SlashCommonTreshold) {
-            throw "/Common Percentage: $SlashCommonPercentage is greater than the allowed treshold: $SlashCommonTreshold"
-        }
+        Write-Host "Retrieving HCX Guest VM Data"
+        $HcxVmGuest = Get-VMGuest -VM $HcxVM
 
-        Write-Host "Retrieved /Common Percentage: $SlashCommonPercentage"
+        $MonitoredDisks = @("/common")
```

Comment:
Disks are physical, I think volume is the noun we're looking for.

Reply:
Actually, since this is how PowerShell model presents it I think we're ok with Disks.